docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+  attack-flow:
+    build:
```

Comment:
Instead of building an image from scratch, please reference the [official project image](https://github.com/center-for-threat-informed-defense/attack-flow/pkgs/container/attack-flow).

Reply:
done, I left the local image usage in comments, let me know if you prefer to remove it